src/main/java/org/gridsuite/study/server/service/securityanalysis/SecurityAnalysisService.java

```diff
@@ -93,6 +95,14 @@ public String getSecurityAnalysisResult(UUID nodeUuid, List<String> limitTypes)
         return result;
     }
 
+    private String getPathFromResultType(SecurityAnalysisResultType resultType) {
+        switch (resultType) {
+            case NMK_CONTINGENCIES : return "nmk-contingencies-result";
```

Comment:
Add path in the enum ?
And remove this method

Reply:
I don't think it's a good idea, we could use this enum for somethings else than just the path